src/sentry/scim/endpoints/teams.py

```diff
@@ -64,7 +64,7 @@
 delete_logger = logging.getLogger("sentry.deletions.api")
 
 
-@extend_schema_serializer(dict)
```

Comment:
this definitely doesn't seem intended -- but it's the equivalent code and I have no idea what the intent was from the original commit